metadata/boundaries/usfs_roadless_inventory.md

```diff
@@ -0,0 +1,73 @@
+# Title
+
+Utah USFS Roadless Inventory
+
+## ID
+
+ae3e16a4-13a8-4af6-a977-b1989ad9880a
+
+## Brief Summary
+
+Polygon dataset of areas in Utah designated as roadless by the 2000 USFS Roadless Inventory.
+
+## Summary
+
+This dataset contains polygon boundaries for inventoried roadless areas managed by the United States Forest Service. Other federally managed areas such as [wilderness areas](https://gis.utah.gov/products/sgid/boundaries/wilderness/) can be found in the [Boundary Data Index](https://gis.utah.gov/products/sgid/boundaries/) available from the SGID.
```

Comment:
```suggestion
This dataset contains polygon boundaries for inventoried roadless areas (IRAs) managed by the United States Forest Service, including designated areas outside of IRAs. Other federally managed areas such as [wilderness areas](https://gis.utah.gov/products/sgid/boundaries/wilderness/) can be found in the [Boundary Data Index](https://gis.utah.gov/products/sgid/boundaries/) available from the SGID.
```
Our data is slightly different that the USFS data in that we include the adjacent "designated areas outside of IRAs"